source/ota_mqtt.c

```diff
@@ -653,7 +653,8 @@ OtaErr_t requestJob_Mqtt( OtaAgentContext_t * pAgentCtx )
      * is calculated from the template and the corresponding parameters. */
     char pJobTopic[ TOPIC_GET_NEXT_BUFFER_SIZE ];
     static uint32_t reqCounter = 0;
-    OtaErr_t result = OTA_ERR_UNINITIALIZED;
+    OtaErr_t result = OtaErrRequestJobFailed;
```

Comment:
`result` could be renamed to `otaError` since we are only storing the error codes and not the results of any ops.